src/sider/senoppfolging/OvingssideLink.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box, Heading, HelpText } from "@navikt/ds-react";
+import React from "react";
+import { EksternLenke } from "@/components/EksternLenke";
+
+const texts = {
+  heading: "Øvingsside for Snart slutt på sykepengene",
+  link: "Se nyeste versjon av svarskjemaet her",
+  helpTextTitle: "Hvorfor kan øvingssiden avvike fra den sykmeldtes svar?",
+  helpText: "Bruker kan ha svart på en tidligere versjon av svarskjemaet.",
+};
+
+const demoUrl =
+  "https://demo.ekstern.dev.nav.no/syk/meroppfolging/snart-slutt-pa-sykepengene";
+
+export default function OvingssideLink() {
+  return (
+    <Box
+      background="surface-default"
+      padding="6"
+      className="flex flex-col gap-4 mb-2"
+    >
+      <div className="flex items-center gap-2">
+        <Heading level="2" size="medium">
+          {texts.heading}
+        </Heading>
+        <HelpText title={texts.helpTextTitle}>{texts.helpText}</HelpText>
```

Comment:
Jeg synes det føltes litt rart med `HelpText` her🤔 Jeg forventet å få opp en forklaring på hva "øving side for snart slutt på sykepenger" var for noe.
Om du ikke har gjort det, så er dette en fin måte å involvere Peter på ved å spørre hva han tenker 😊

Reply:
Godt innspill! Jeg kan høre med Peter 😊

Reply:
Jeg tenker også at den helptexten står bedre sammen med lenke-teksten (Se nyeste osv, men obs bruker kan ha svart på en tidligere versjon) og ikke til overskriften. Men ja, hør med Peter :)

Reply:
Godt poeng! Da fjerner jeg hjelpeteksten. Virker som den kan forvirre mer enn den hjelper.